integration/sync_test.go

```diff
@@ -202,8 +202,9 @@ func (s *SyncSuite) TestDocker2DirUntagged(c *check.C) {
 	defer os.RemoveAll(tmpDir)
 
 	// FIXME: It would be nice to use one of the local Docker registries instead of needing an Internet connection.
-	image := "alpine"
-	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s", image))
+	registry := "k8s.gcr.io"
+	image := "pause"
+	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s/%s", registry, image))
```

Comment:
This now results in `… sync --src docker pause`, not what was intended.

Reply:
Thanks for catching that.